tests/test_setup.sh

```diff
@@ -0,0 +1,63 @@
+cd tests
```

Comment:
Good, but can we give this script a more meaningful name - tests/create_katsu_datasets.sh or something?